src/mainwindow.cpp

```diff
@@ -234,6 +236,23 @@ void MainWindow::setup_ActionsMenu_Actions()
     setup_Action(MOVE_RIGHT, new QAction(tr("Move Tab &Right"), settingOwner),
                  MOVE_RIGHT_SHORTCUT, consoleTabulator, SLOT(moveRight()), menu_Actions, data);
 
+    menu_Actions->addSeparator();
+    data.setValue(checkHasIndexedTab);
+
+    const QString textBase = tr("Tab");
+    QSignalMapper *signalMapper = new QSignalMapper(this);
+    QObject::connect(signalMapper, SIGNAL(mapped(int)), consoleTabulator, SLOT(switchTab(int)));
+    QObject::connect(signalMapper, SIGNAL(mapped(int)), this, SLOT(gotshit(int)));
```

Comment:
Is it possible to use the new Qt5 syntax for `connect()` instead of the SIGNAL-SLOT syntax?

Reply:
I've removed the signalMapper since it turns out it's been deprecated.